### PR TITLE
Don't hardcode serving as the repo when setting highest semver

### PR DESCRIPTION
**Pull request by Cali0707**
Fixes errors like:
```
Setting latest release to highest semver
exit status 22
**************************************************
**** ERROR: error settomg v1.13.0 to 'latest' ****
************************************************** 
```
in the release jobs
# Changes

- Don't hardcode serving, use env variables instead

### CODE REVIEW

**Comment by dsimansk** [comment]
/approve
/lgtm

For Productivity WG to check.
/hold

File: release.sh [hunk 1]
```diff
@@ -675,7 +675,7 @@ function set_latest_to_highest_semver() {
     abort "cannot get relase id from github"
   fi
   
-  hub_tool api --method PATCH "/repos/knative/serving/releases/$release_id" \
+  hub_tool api --method PATCH "/repos/${ORG_NAME}/${REPO_NAME}/releases/$release_id" \
     -F make_latest=true > /dev/null || abort "error settomg $last_version to 'latest'"
```
Comment (krsna-m) [inline release.sh:679]:
```suggestion
    -F make_latest=true > /dev/null || abort "error setting $last_version to 'latest'"
```

**Pull request merged**